index.js

```diff
@@ -15,21 +15,12 @@ const fastifySlonik = async (fastify, options) => {
     fastify.log.error(err)
   }
 
-  async function transaction (connection) {
-    return pool.query(connection)
-  }
-
-  const db = {
-    connect: pool.connect.bind(pool),
-    pool: pool,
-    query: pool.query.bind(pool),
-    transaction: transaction.bind(pool),
-    exists: pool.exists.bind(pool),
-    one: pool.one.bind(pool)
-  }
-
-  fastify.decorate('slonik', db)
+  fastify.decorate('slonik', pool)
   fastify.decorate('sql', sql)
```

Comment:
thoughts on making slonik the root and moving `sql` to be under it rather than having two slonik pieces imported - it's funny how we have `slonik` and then sloniks `sql` (and anything else we might want to bring it as it's introduced) 

is this a case where we can decorate with a namespace so that we can do `fastify.slonik` and `fastify.slonik.sql` and alias as needed?  it's certainly more clear and keeps the concerns together but what do you think?

Reply:
Agreed, I think that makes sense. Updated.